public/video-ui/styles/abstracts/_mixins.scss

```diff
@@ -0,0 +1,21 @@
+@mixin buttonClass($colorA, $colorB) {
```

Comment:
last comment... Can `colorA` and `colorB` be more expressive in their naming? as its not too clear which is used where.